dohq_artifactory/admin.py

```diff
@@ -377,7 +377,7 @@ def _read_response(self, response):
         """
         self.name = response["key"]
         self.description = response.get("description")
-        self.layoutName = response.get("repoLayoutRef")
```

Comment:
Oh, yes. I don't known why we've called this property `layoutname` instead of really name... But may be we can keep this name and add additional one for backward compatibility reason? 

We can just use two the same property and leave short comment about this. 
Or we can wrap `layoutName` to `@property` and use [warning](https://docs.python.org/3/library/warnings.html#warnings.warn) to warn users about future removal.

You choose, for me both are fine :)

Reply:
But... it's easy to fix it after library update. LGTM

Reply:
Hopefully correctly addressed this in #152